packages/buildbucket-dart/lib/src/generated/go.chromium.org/luci/buildbucket/proto/build.pb.dart

```diff
@@ -0,0 +1,2650 @@
+///
+//  Generated code. Do not modify.
+//  source: go.chromium.org/luci/buildbucket/proto/build.proto
+//
+// @dart = 2.12
+// ignore_for_file: annotate_overrides,camel_case_types,constant_identifier_names,directives_ordering,library_prefixes,non_constant_identifier_names,prefer_final_fields,return_of_invalid_type,unnecessary_const,unnecessary_import,unnecessary_this,unused_import,unused_shown_name
+
+import 'dart:core' as $core;
+
+import 'package:fixnum/fixnum.dart' as $fixnum;
+import 'package:protobuf/protobuf.dart' as $pb;
+
+import 'builder_common.pb.dart' as $0;
+import '../../../../google/protobuf/timestamp.pb.dart' as $1;
+import 'step.pb.dart' as $2;
+import 'common.pb.dart' as $3;
+import '../../../../google/protobuf/duration.pb.dart' as $4;
+import '../../../../google/protobuf/struct.pb.dart' as $5;
+import 'task.pb.dart' as $6;
+
+import 'common.pbenum.dart' as $3;
+import 'build.pbenum.dart';
+
+export 'build.pbenum.dart';
+
+class Build_Input extends $pb.GeneratedMessage {
```

Comment:
Seems like protoc doesn't copy comments into the Dart code. Would it be ok to leave the chromium protos in this repo so it's easier to find the documentation?

Reply:
It will require some extra work to copy all the dependencies as they are coming from three different repos. I'll try to find a good way to copy the files without impacting too much in the maintenance.